mmv1/third_party/terraform/services/datastream/resource_datastream_stream_test.go

```diff
@@ -33,7 +33,7 @@ func TestAccDatastreamStream_update(t *testing.T) {
 				ResourceName:            "google_datastream_stream.default",
 				ImportState:             true,
 				ImportStateVerify:       true,
-				ImportStateVerifyIgnore: []string{"create_without_validation", "stream_id", "location", "desired_state", "labels", "terraform_labels"},
+				ImportStateVerifyIgnore: []string{"gtid", "binaryLogPosition", "create_without_validation", "stream_id", "location", "desired_state", "labels", "terraform_labels"},
```

Comment:
Is there a reason are are adding these to ImportStateVerifyIgnore?

Reply:
Tests were failing because of these so ignored them. now removed them after fixing the code.